cypress-tests/cypress/support/redirectionHandler.js

```diff
@@ -395,6 +395,15 @@ function upiRedirection(
 }
 
 function verifyReturnUrl(redirection_url, expected_url, forward_flow) {
+  const urlParams = new URLSearchParams(redirection_url.search);
+  const paymentStatus = urlParams.get('status');
+
+  // Check for valid statuses
+  if (paymentStatus !== 'succeeded' && paymentStatus !== 'processing') {
```

Comment:
we do have one more status called as `partially_captured`
```suggestion
  if (paymentStatus !== 'succeeded' && paymentStatus !== 'processing' && paymentStatus !== 'partially_captured') {
```